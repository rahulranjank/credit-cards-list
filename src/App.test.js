import React from 'react'
import {render, cleanup} from '@testing-library/react'
import App from './router/AppRouter';

describe('Take a snapshot', () => {
  afterEach(cleanup)
    it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
   })
});

