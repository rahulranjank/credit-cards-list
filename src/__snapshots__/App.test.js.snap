// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Take a snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <main
    class="container"
  >
    <div>
      <div>
        <div>
          <div>
            <div>
              List of Credit Cards
            </div>
            <br />
            <a
              href="/add"
            >
              <div>
                <button
                  class="add-credit-card-button"
                >
                   Add Credit Card
                </button>
              </div>
            </a>
          </div>
          <div>
            <p>
              No card available. Please add some cards.
            </p>
          </div>
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
